README.md

```diff
@@ -138,25 +137,30 @@ For supported JavaScript runtimes, please consult [RUNTIMES.md](RUNTIMES.md).
 
 ## Available environments
 
-Most customer accounts run on LaunchDarkly's commercial (default) environment. Customers on other environments can specify the `--server-url` argument to connect to the appropriate environment. For example, customers on LaunchDarkly's Federal environment should specify `--server-url https://app.launchdarkly.us` when starting their MCP server.
+Most customer accounts run on LaunchDarkly's commercial (default) environment. Customers on other environments can specify the `--server-url` argument to connect to the appropriate environment. For example, customers on LaunchDarkly's Federal environment should specify the `--server-url https://app.launchdarkly.us` argument when starting their MCP server.
 
-| #   | Server URL                        | Environment          |
-| --- | --------------------------------- | -------------------- |
-| 0   | `https://app.launchdarkly.com`    | Commercial (Default) |
-| 1   | `https://app.launchdarkly.us`     | Federal              |
-| 2   | `https://app.eu.launchdarkly.com` | EU                   |
-
-# Development
-
-## Maturity
-
-This SDK is in beta, and there may be breaking changes between versions without a major version update. Therefore, we recommend pinning usage
-to a specific package version. This way, you can install the same version each time without breaking changes unless you are intentionally
-looking for the latest version.
+| Environment          | Server URL                        |
+| -------------------- | --------------------------------- |
+| Commercial (Default) | `https://app.launchdarkly.com`    |
+| Federal              | `https://app.launchdarkly.us`     |
+| EU                   | `https://app.eu.launchdarkly.com` |
 
 ## Contributions
 
 While we value open-source contributions to this SDK, this library is generated programmatically. Any manual changes added to internal files will be overwritten on the next generation. 
 We look forward to hearing your feedback. Feel free to open a PR or an issue with a proof of concept and we'll do our best to include it in a future release. 
 
-### SDK Created by [Speakeasy](https://www.speakeasy.com/?utm_source=@launchdarkly/mcp-server&utm_campaign=typescript)
+## About LaunchDarkly
```

Comment:
This is the standard footer content from other repos -- except that I updated the link URLs to match our new site structure.